feature-utils/poly-look/storybook/stories/react-components/sideSlider.stories.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react";
+import SideSlider from "../../../src/react-components/overlays/sideSlider.jsx";
+import "../../../src/css/index.js";
+import "./fontFamily.css";
+import "./demo.css";
+
+export default {
+  component: SideSlider,
+  title: "Visuals/Organisms/SideSlider",
+};
+
+const Template = (args) => {
+  const [open, setOpen] = useState(false);
+
+  const SliderComponent = (props) => {
+    let buttonClick = (ev) => {
+      ev.stopPropagation();
+      console.log("click inside comp, no close");
```

Comment:
is this log meant to stay here?

Reply:
Personally, I don't have an issue with leaving `console.logs` in stories. There is a nicer way to handle this using [actions](https://storybook.js.org/docs/react/essentials/actions), but since didn't do that, I figured a log is good enough 😅 I can remove it and do it the nicer way.

Reply:
No no if you are good with it then all good. Was just making sure it wasn't forgotten :)